{"abstract":[{"text":"Indicates that a timeout occurred during peripheral connection.","type":"text"}],"metadata":{"externalID":"s:11BlueConnect27BleCentralManagerProxyErrorO17connectionTimeoutyA2CmF","modules":[{"name":"BlueConnect"}],"title":"BleCentralManagerProxyError.connectionTimeout","symbolKind":"case","fragments":[{"kind":"keyword","text":"case"},{"kind":"text","text":" "},{"text":"connectionTimeout","kind":"identifier"}],"roleHeading":"Case","role":"symbol"},"identifier":{"interfaceLanguage":"swift","url":"doc:\/\/BlueConnect\/documentation\/BlueConnect\/BleCentralManagerProxyError\/connectionTimeout"},"sections":[],"kind":"symbol","hierarchy":{"paths":[["doc:\/\/BlueConnect\/documentation\/BlueConnect","doc:\/\/BlueConnect\/documentation\/BlueConnect\/BleCentralManagerProxyError"]]},"schemaVersion":{"minor":3,"major":0,"patch":0},"primaryContentSections":[{"kind":"declarations","declarations":[{"platforms":["macOS"],"tokens":[{"text":"case","kind":"keyword"},{"text":" ","kind":"text"},{"text":"connectionTimeout","kind":"identifier"}],"languages":["swift"]}]},{"kind":"content","content":[{"text":"Discussion","level":2,"anchor":"discussion","type":"heading"},{"type":"paragraph","inlineContent":[{"text":"This error is thrown when a peripheral connection attempt exceeds the allowed time limit.","type":"text"}]}]}],"variants":[{"traits":[{"interfaceLanguage":"swift"}],"paths":["\/documentation\/blueconnect\/blecentralmanagerproxyerror\/connectiontimeout"]}],"references":{"doc://BlueConnect/documentation/BlueConnect":{"type":"topic","url":"\/documentation\/blueconnect","title":"BlueConnect","abstract":[],"kind":"symbol","role":"collection","identifier":"doc:\/\/BlueConnect\/documentation\/BlueConnect"},"doc://BlueConnect/documentation/BlueConnect/BleCentralManagerProxyError":{"role":"symbol","type":"topic","title":"BleCentralManagerProxyError","navigatorTitle":[{"kind":"identifier","text":"BleCentralManagerProxyError"}],"abstract":[{"text":"An enumeration representing various errors that can occur in the ","type":"text"},{"code":"BleCentralManagerProxy","type":"codeVoice"},{"type":"text","text":"."}],"kind":"symbol","identifier":"doc:\/\/BlueConnect\/documentation\/BlueConnect\/BleCentralManagerProxyError","fragments":[{"text":"enum","kind":"keyword"},{"text":" ","kind":"text"},{"kind":"identifier","text":"BleCentralManagerProxyError"}],"url":"\/documentation\/blueconnect\/blecentralmanagerproxyerror"},"doc://BlueConnect/documentation/BlueConnect/BleCentralManagerProxyError/connectionTimeout":{"kind":"symbol","role":"symbol","url":"\/documentation\/blueconnect\/blecentralmanagerproxyerror\/connectiontimeout","title":"BleCentralManagerProxyError.connectionTimeout","type":"topic","abstract":[{"type":"text","text":"Indicates that a timeout occurred during peripheral connection."}],"identifier":"doc:\/\/BlueConnect\/documentation\/BlueConnect\/BleCentralManagerProxyError\/connectionTimeout","fragments":[{"text":"case","kind":"keyword"},{"kind":"text","text":" "},{"kind":"identifier","text":"connectionTimeout"}]}}}